kvm/src/lib.rs

```diff
@@ -44,6 +45,37 @@ pub use kvm_bindings::KVM_API_VERSION;
 /// Taken from Linux Kernel v4.14.13 (arch/x86/include/asm/kvm_host.h)
 pub const MAX_KVM_CPUID_ENTRIES: usize = 80;
 
+// Returns a `Vec<T>` with a size in bytes at least as large as `size_in_bytes`.
```

Comment:
I would suggest to move `vec_with_size_in_bytes` and `vec_with_array_field` in a helper class.

Reply:
I wouldn't create a helper class only for these two functions as I do not see an advantage of having said helper class. What do you think would be the benefit of having a helper class?